framework/packages/abstract-core/src/native/version_control.rs

```diff
@@ -286,14 +304,15 @@ impl ModuleConfiguration {
 
     pub fn from_storage(storage: &dyn Storage, module: &ModuleInfo) -> Self {
```

Comment:
These are at least 3 load operations for a single module. Can we combine their information into a single data store so we only have to load once?

Reply:
Yep, changed state to have distinct storage for "default" items for module config as well!